demo/workflow/workflow.md

```diff
@@ -0,0 +1,722 @@
+
+<img width="1260" alt="3" src="https://user-images.githubusercontent.com/88045217/233478647-a62fccb0-1e87-4542-b089-2091155c553c.png">
+
+### **Bruce Wayne** : Which is?
+
+<img width="1260" alt="3" src="https://user-images.githubusercontent.com/88045217/233477213-f6d99be5-f669-4344-8c1e-123f063db3de.png">
+
+### **Henri Ducard (Ra's al Ghul)** : Legend, Mr. Wayne!
+
+<img width="1260" alt="3" src="https://user-images.githubusercontent.com/88045217/233479006-9fa20a7e-1a13-41d1-9f97-2345fcd85c69.png">
+
+Ra’s al Ghul understands that an SBOM alone can be forgotten but if you combine
+it with GUAC, you can start to get a greater understanding of your own software
+supply chain environment.
+
+In this demo, we will go through the process of ingesting an SBOM and letting
+GUAC expand our horizons on what we know about our environment autonomously!
+
+## Requirements
+
+- go
+- Kubernetes 1.19+ ([k3s](https://github.com/k3s-io/k3s/),
+  [minikube](https://github.com/kubernetes/minikube) or
+  [colima](https://github.com/abiosoft/colima) (or another k8s service of your
+  choice))
+- [Helm v3.9.4+](https://helm.sh/)
+
+## Clone GUAC
+
+If you haven't already, clone GUAC to a local directory:
+
+```bash
+git clone https://github.com/guacsec/guac.git
+```
+
+Also, clone GUAC data, this is used as test data for this demo.
+
+```bash
+git clone https://github.com/guacsec/guac-data.git
+```
+
+The rest of the demo will assume you are in the GUAC directory
+
+```bash
+cd guac
+```
+
+## Building the GUAC binaries
+
+Build the GUAC binaries using the `make` command.
+
+```bash
+make
+```
+
+## Installing GUAC via Helm Chart
+
+Please refer to the [INSERT GUAC Helm Install Guide] to have a running instance.
+
+Once that process is completed, you will see the following pods running (the pod
+names are auto-generated so they might differ):
+
+```bash
+default       osv-certifier-c7c67d89d-f5ndx             1/1     Running     0             11m
+default       guac-nats-box-774449cffc-vdkmx            1/1     Running     0             11m
+default       collectsub-855c4578f6-69nfx               1/1     Running     0             11m
+default       graphql-server-589c77fdb4-hvprm           1/1     Running     0             11m
+default       guac-nats-0                               3/3     Running     0             11m
+default       depsdev-collector-8698956798-x74vr        1/1     Running     2 (11m ago)   11m
+default       oci-collector-7ff77d9b7-k24zk             1/1     Running     2 (11m ago)   11m
+default       ingestor-5c75564464-s5v2s                 1/1     Running     2 (11m ago)   11m
+default       ingest-guac-data-98rfq                    0/2     Completed   1             11m
```

Comment:
hmmm is this ingesting data?

Reply:
this is going to be removed from the helm deployment. Removed it from the demo